lib/internal/process/task_queues.js

```diff
@@ -99,16 +103,30 @@ function processTicksAndRejections() {
   setHasRejectionToWarn(false);
 }
 
+function nextTickDefaultCallback(resolve) {
+  resolve();
+}
+
 // `nextTick()` will not enqueue any callback when the process is about to
 // exit since the callback would not have a chance to be executed.
 function nextTick(callback) {
+  let promise;
+
+  if (!arguments.length) {
+    const defered = createDeferredPromise();
+    promise = defered.promise;
+    const { resolve } = defered;
+    callback = nextTickDefaultCallback.bind(undefined, resolve);
```

Comment:
I think this could be simplified without the destructuring assignment above this line along with:

```suggestion
    callback = nextTickDefaultCallback.bind(undefined, deferred.resolve);
```

Reply:
done.